### PR TITLE
Installation guide for configuring logger

### LINKED ISSUE #1104: Feature Request: Add documentation for configuring logger
**Is your feature request related to a problem? Please describe.**
Currently, our logger has support for both, different log levels and colourised logging. But there is no central documentation on as to what are the different options that can be used for `log level`, what differences they have, and how to set them up.
 
**Describe the solution you'd like**
Add documentation for the same in the respective markdown files.

**Describe alternatives you've considered**
NA

**Approach to be followed (optional)**
Update documetation

**Additional context**
NA

### PR DESCRIPTION
**Pull request by KrutikaBhatt**
**What kind of change does this PR introduce?**
This PR introduces changes in `.env.sample file` for logger configuration. The description to setup the logger values in .env file has been added to  `INSTALLATION.md`

**Issue Number: #1104**

This PR fixes #1104 

**Did you add tests for your changes? No**

**If relevant, did you update the documentation? No**

**Summary**
Currently, our logger has support for both, different log levels and colorized logging. But there is no documentation on as to what are the different options that can be used for log level, what differences they have, and how to set them up. Moreover, the logger values used in the application are referred from the `.env` file. There is no mention on how to set them and the different values it can be configured. Thus changes are made in .env.sample and also in INSTALLATION.md

**Does this PR introduce a breaking change? No**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)? Yes**

### CODE REVIEW

**Comment by KrutikaBhatt** [comment]
I am not sure if the changes should be made in central DOCUMENTATION for logger configuration since it is more developer and application relevant and is required during the installation and setup of `talawa-api` only. But I would like to get some feedback on this. If there are any other suggested modifications, I would be glad to make those changes.

**Comment by jadeharr** [comment]
@KrutikaBhatt Please make sure that all your checks have been passed.

**Comment by KrutikaBhatt** [comment]
The failing test is Checking for unauthorized file changes. Is .env.sample or INSTALLATION.md an unauthorized file?

**Review by noman2002 — APPROVED** [review]
Looks good to me.

File: INSTALLATION.md [hunk 1]
```diff
@@ -261,10 +263,36 @@ The MAIL_USERNAME and MAIL_PASSWORD parameters are required to enable an app to
 1. Under `Signing in to Google` section select `App Passwords`.
 1. Click on `Select app` section and choose `Other(Custom name)`, enter `talawa` as the custom name and press `Generate` button.
 1.  Copy the 16 character generated app password to the variable named `MAIL_PASSWORD` in `.env` file.
-1.  Copy you usual gmail address to the variable named `MAIL_USERNAME` in `.env` file.
+1.  Copy your usual gmail address to the variable named `MAIL_USERNAME` in `.env` file.
 
 For more info refer to this [Google Answer](https://support.google.com/accounts/answer/185833).
 
+### Setting up COLORIZE_LOGS in .env file
```
Comment (palisadoes) [inline INSTALLATION.md:270]: Is this optional?  If so, this must be clearly stated. We have too many steps already.

File: INSTALLATION.md [hunk 2]
```diff
@@ -261,10 +263,36 @@ The MAIL_USERNAME and MAIL_PASSWORD parameters are required to enable an app to
...
+The parameter `COLORIZE_LOGS` is a boolean field and can be set to true or false. It customizes the log colorization formats displayed in console. You can set the value in `.env` file as
```
Comment (palisadoes) [inline INSTALLATION.md:271]: Regenerate the table of contents with this information. Make sure the indentation is correct

File: INSTALLATION.md [hunk 3]
```diff
@@ -261,10 +263,36 @@ The MAIL_USERNAME and MAIL_PASSWORD parameters are required to enable an app to
...
+### Setting up COLORIZE_LOGS in .env file
+The parameter `COLORIZE_LOGS` is a boolean field and can be set to true or false. It customizes the log colorization formats displayed in console. You can set the value in `.env` file as 
+```
+COLORIZE_LOGS = false
+```
+If the parameter value is set to `true`, you should be able to see colorized logs in console, or else logs will display in the console's default simple format.
+
+![Colorized logs in console](./image/colorize-logs.jpg)
+
+### Setting up LOG_LEVEL in .env file
```
Comment (palisadoes) [inline INSTALLATION.md:279]: Regenerate the table of contents with this information. Make sure the indentation is correct

**Review by palisadoes — CHANGES_REQUESTED** [review]

File: INSTALLATION.md (hunk 1 above)
Comment (KrutikaBhatt) [reply INSTALLATION.md]: Yes sir, I have made these changes in recent commits

File: INSTALLATION.md (hunk 3 above)
Comment (KrutikaBhatt) [reply INSTALLATION.md]: Added this segment in table of contents

File: INSTALLATION.md (hunk 2 above)
Comment (KrutikaBhatt) [reply INSTALLATION.md:273]: Added this segment in the table of contents

**Comment by KrutikaBhatt** [comment]
I was having university exams. Apologize on my behalf for the delay in implementing these small changes.

**Comment by DMills27** [comment]
Are your exams still going on?

**Comment by palisadoes** [comment]
We need to consider removing colorized logs. When the github actions fail I think the colorizations and clear screens that happen are making it difficult to track.

**Comment by KrutikaBhatt** [comment]
I guess the GitHub action fails for modifying the .env.sample file. Does the problem of colorization and clear screen happen while running the application too? Colorized logs are just an added feature; they can surely be considered to get removed but I don't that's causing this issue. Any ideas on this ?

**Comment by KrutikaBhatt** [comment]
> Are your exams still going on?

Would be free by today. Thanks for waiting up for me

**Comment by palisadoes** [comment]
> I guess the GitHub action fails for modifying the .env.sample file. Does the problem of colorization and clear screen happen while running the application too? Colorized logs are just an added feature; they can surely be considered to get removed but I don't that's causing this issue. Any ideas on this ?

No it's not causing the issue, but I've noticed it creating difficulties scrolling through the errors.

**Comment by KrutikaBhatt** [comment]
> > I guess the GitHub action fails for modifying the .env.sample file. Does the problem of colorization and clear screen happen while running the application too? Colorized logs are just an added feature; they can surely be considered to get removed but I don't that's causing this issue. Any ideas on this ?
> 
> No it's not causing the issue, but I've noticed it creating difficulties scrolling through the errors in GitHub Actions.

Ok, yes. We can consider removing them. This just will require just if-else changes in `logger.ts` and an extra configuration can be avoided in INSTALLATION.md too.

**Comment by KrutikaBhatt** [comment]
@palisadoes, any feedback on this ?

**Comment by palisadoes** [comment]
Please fix the conflicting files

**Review by noman2002 — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**